devices/google-blueline/kernel-mainline/default.nix

```diff
@@ -0,0 +1,48 @@
+{
+  mobile-nixos
+, fetchFromGitHub
+, fetchpatch
+, ...
+}:
+
+mobile-nixos.kernel-builder rec {
+  version = "5.13.0-rc2";
+  configfile = ./config.aarch64;
+
+  #src = /Users/samuel/tmp/linux/linaro-pixel3;
+
+  # FIXME: apply patchsets directly on top of mainline
+  src = fetchFromGitHub {
+    # Mirror of http://git.linaro.org/people/vinod.koul/kernel.git/commit/?h=pixel/dsc_rfc&id=dcf128e83ddd25e461a5080a349de5a0552eed32
```

Comment:
Why mirror? To use `fetchFromGitHub` rather than a lengthy clone.

And **anyway** the goal is to just use the mainline kernel and add patches... So this will not stay... Though maybe the patch will come from that mirror as a span of commits.